src/test/e2e/variable_test.go

```diff
@@ -45,6 +45,10 @@ func TestBundleVariables(t *testing.T) {
 	require.Contains(t, stderr, "This fun-fact demonstrates precedence: The Red Dragon is the national symbol of Wales")
 	require.Contains(t, stderr, "shared var in output-var pkg: burning.boats")
 	require.Contains(t, stderr, "shared var in receive-var pkg: burning.boats")
+
+	_, stderr = deployCustom(t, "deploy "+bundlePath+" --set ANIMAL=Longhorns --set COUNTRY=Texas --confirm -l=debug")
```

Comment:
What is `+bundlePath+"?

Reply:
the bundle tarball path, updated the name to that (bundleTarballPath) for clarity